internal/services/fluidrelay/fluid_relay_servers_resource_test.go

```diff
@@ -27,7 +27,6 @@ func TestAccFluidRelay_basic(t *testing.T) {
 				check.That(data.ResourceName).ExistsInAzure(f),
 				check.That(data.ResourceName).Key("frs_tenant_id").IsUUID(),
 				check.That(data.ResourceName).Key("primary_key").Exists(),
-				check.That(data.ResourceName).Key("orderer_endpoints.0").Exists(),
```

Comment:
This check was working until Jan 4th, is this a correct fix, or are we missing a service change/bug here?

Reply:
the service team has deprecated the orderer and storage endpoint pair. it should be safe to remove this check in the test and we may also need to update our document for them.

Reply:
here is a document https://learn.microsoft.com/en-us/azure/azure-fluid-relay/concepts/version-compatibility